queries.md

```diff
@@ -2,20 +2,42 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+SELECT * FROM customers where city = 'London';
+
+
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+SELECT * FROM customers where PostalCode = '1010';
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+SELECT * FROM [Suppliers] where SupplierId = '11'
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 
+SELECT * FROM [Orders]
+ORDER BY OrderDate DESC;
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 
+SELECT * FROM [Suppliers] where Length(SupplierName) > 20
+
 ## find all customers that include the word "market" in the name. Should return 4 records.
 
+SELECT * FROM [Customers] where CustomerName LIKE '%market%'
+
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 
+INSERT INTO Customers (CustomerName, ContactName, Address, City, PostalCode, Country)
+VALUES ('_"The Shire"_','_"Bilbo Baggins"_','_"1 Hobbit-Hole"_ in _"Bag End"_','','_"111"_','_"Middle Earth"_');
```

Comment:
What do the underscores do? I'm genuinely curious!

Reply:
Actually, I think I know what is going on here. The instructions was:
```
## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
```
This, and any file that ends with `.md`, is a _Markdown_ file. It's kinda like HTML-light that deals much more on formatting and structure. For example, `"This is _italicized_."` will render to "This is _italicized_." Let me know if you want to know more about Markdown.

But long story short, the underscores aren't part of the instructions. It's really this:

---

## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.